.github/workflows/ci.yml

```diff
@@ -142,13 +137,36 @@ jobs:
     - name: make
       run: make -j4
     - name: get cpu info
-      run: ./util/opensslwrap.sh version -c
+      run: |
+        cat /proc/cpuinfo
+        ./util/opensslwrap.sh version -c
     - name: make test
       run: .github/workflows/make-test
     - name: save artifacts
       uses: actions/upload-artifact@v4
       with:
-        name: "ci@self-hosted-${{ matrix.os }}"
+        name: "ci@linux-arm64"
+        path: artifacts.tar.gz
+  
+  BSD-x86_64:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: cross-platform-actions/action@v0.26.0
+      with:
+        operating_system: freebsd
+        image_url: https://github.com/quarckster/freebsd-builder/releases/download/25-01-02-2/freebsd-13.4-x86-64.qcow2
```

Comment:
Do we want to depend on this image file hosted in your repo? Can't we use a default freebsd image from cross-platform-actions?

Reply:
I'd prefer installing them unless such thing would take too many cycles. We are installing things in other CI runs as well.

Reply:
Done.